.github/workflows/copybara.yaml

```diff
@@ -0,0 +1,22 @@
+# This script create a Pull request when there is push to copybara branch
+
+name: create-pr-for-copybara
```

Comment:
This should be the human readable name. No dashes required.

Suggestion:

`name: Create Copybara PR`